package.json

```diff
@@ -29,6 +29,9 @@
     "tape": "^4.6.0"
   },
   "dependencies": {
-    "plist": "^2.0.1"
+    "async": "^2.1.4",
+    "iconutil": "^1.0.1",
+    "sha1": "^1.1.1",
```

Comment:
This dependency has sub-dependencies, but it's built into node.

~~~ javascript
const crypto = require('crypto')
const sha1 = (message) => crypto.createHash('sha1').update(message).digest().toString('hex')
sha1('alice') // 522b276a356bdf39013dfabea2cd43e141ecc9e8
~~~